contributed_definitions/nyaml/NXmpes_liquid.yaml

```diff
@@ -4,92 +4,15 @@
-          enumeration: ["x-ray","ultraviolet", "visible light"]
-      (NXbeam): 
-        distance(NX_NUMBER):
-          doc: "Distance of the point of evaluation of the beam from the sample surface."
-        incident_energy(NX_FLOAT):
-          unit: NX_ENERGY
-          doc: "In the case of a monchromatic beam this is the scalar energy. Several other use cases are permitted, depending on the presence of other incident_energy_X fields. In the case of a polychromatic beam this is an array of length m of energies, with the relative weights in incident_energy_weights. In the case of a monochromatic beam that varies shot-to-shot, this is an array of energies, one for each recorded shot. Here, incident_energy_weights and incident_energy_spread are not set. In the case of a polychromatic beam that varies shot-to-shot, this is an array of length m with the relative weights in incident_energy_weights as a 2D array. In the case of a polychromatic beam that varies shot-to-shot and where the channels also vary, this is a 2D array of dimensions nP by m (slow to fast) with the relative weights in incident_energy_weights as a 2D array. Note, variants are a good way to represent several of these use cases in a single dataset, e.g. if a calibrated, single-value energy value is available along with the original spectrum from which it was calibrated."
-        incident_energy_spread(NX_NUMBER):
-          exists: recommended
-          unit: NX_ENERGY
-          doc: "The energy spread FWHM for the corresponding energy(ies) in incident_energy. In the case of shot-to-shot variation in the energy spread, this is a 2D array of dimension nP by m (slow to fast) of the spreads of the corresponding energy in incident_energy."
-        incident_polarization(NX_NUMBER):
-          exists: recommended
-          doc: "Incident polarization specified as a Stokes vector."
-          \@units(NX_CHAR):
-            doc: "The units for this observable are not included in the NIAC list. Responsibility on correct formatting and parsing is handed to the user by using ‘NX_ANY’. Correct parsing can still be implemented by using this attribute. Fill with: The unit unidata symbol if the unit has one (Example: ‘T’ for the unit of magnetic flux density tesla). The unit unidata name if the unit has a name (Example: ‘farad’ for capacitance). A string describing the units according to unidata unit operation notation, if the unit is a complex combination of named units and does not have a name. Example: for lightsource brilliance (SI) ‘1/(s.mm2.mrad2)’. Here: SI units are ‘V2/m2’."
       (NXelectronanalyser):
-        description(NX_CHAR):
-          doc: "Free text description of the type of detector."
-        energy_resolution(NX_FLOAT):
-          doc: "Energy resolution of the analyser with the current setting. May be linked from a NXcalibration."
-        fast_axes(NX_CHAR):
```

Comment:
`slow_axes` and `fast_axes` were set to optional instead of recommended (compared to NXmpes). Is this intentionally? Generally sub-definitions should not lift requirements but rather make them more specific. It is still ok for recommended vs optional but it is not best practice and we should have a good reason to change it here.